support-frontend/assets/pages/[countryGroupId]/oneTimeCheckout.tsx

```diff
@@ -206,6 +206,25 @@ function OneTimeCheckoutComponent({
 
 	const formRef = useRef<HTMLFormElement>(null);
 
+	const displayValidity = (
```

Comment:
As a second point to this, it does not really matter - but would `onInvalid` or `setInvalid` be a more descriptive name? Trying to work out what `display` is referencing?

Reply:
I had it as `setInvalid` but thought was too close in naming to a Set<State> calls. hence used the word `displayValidity` as it checks the validity for a field and then displays the error message. Do you think I shoul dnot worry about using set in functions alongside SetState?

Reply:
> checks the validity for a input text field and then displays the error message

This feels like part of it - what it does is check the validity and then conditionally show or hide the error message based on that.

Maybe just `validate` could work?

Reply:
Yes thats more concise, done, thanks.